library/Icingadb/Widget/BaseListItem.php

```diff
@@ -40,6 +40,10 @@ public function __construct($item, BaseItemList $list)
 
         $this->addAttributes($this->baseAttributes);
 
+        if (isset($this->item->icon_image->icon_image)) {
+            $this->list->setHasIconImages(true);
+        }
+
```

Comment:
This can't be here, put this back into `StateListItem`. You can't expect every list item to have an `icon_image` relation.